micro-rdk/src/common/webrtc/api.rs

```diff
@@ -299,47 +300,92 @@ impl WebRtcSignalingChannel {
         &mut self,
         uuid: String,
     ) -> Result<(), WebRtcError> {
-        let answer = AnswerResponse {
-            uuid,
-            stage: Some(answer_response::Stage::Error(AnswerResponseErrorStage {
-                status: Some(Status {
-                    code: Code::ResourceExhausted.into(),
-                    message: "too many active connections".to_string(),
-                    ..Default::default()
-                }),
-            })),
-        };
-
-        if let Err(e) = self.signaling_tx.send_message(answer).await {
-            log::error!("error sending signaling message: {:?}", e);
-            Err(WebRtcError::SignalingDisconnected())
-        } else {
-            log::warn!("too many active connections");
-            Ok(())
+        match self.signaling.as_mut() {
```

Comment:
The app signaling case is now the `Left` case, and the code there hasn't changed. The `Right` case is local signaling and is new code. That goes for many of the following methods.

I have made no effort to DRY things up between the app signaling and the local signaling case, but will do so in a follow-up